var/spack/repos/builtin/packages/xproto/package.py

```diff
@@ -26,17 +26,21 @@
 
 
 class Xproto(Package):
-    """The Xorg protocol headers provide the header files required to
-       build the system, and to allow other applications to build against
-       the installed X Window system."""
-    homepage = "http://www.x.org/"
-    url      = "https://www.x.org/archive//individual/proto/xproto-7.0.29.tar.gz"
+    """X Window System Core Protocol.
 
-    version('7.0.29', '16a78dd2c5ad73011105c96235f6a0af')
+    This package provides the headers and specification documents defining
+    the X Window System Core Protocol, Version 11.
+
+    It also includes a number of headers that aren't purely protocol related,
+    but are depended upon by many other X Window System packages to provide
+    common definitions and porting layer."""
 
-    depends_on("xorg-util-macros")
```

Comment:
Hmm, now I'm seeing warning messages like:
```
Package xorg-macros was not found in the pkg-config search path.
Perhaps you should add the directory containing `xorg-macros.pc'
to the PKG_CONFIG_PATH environment variable
No package 'xorg-macros' found
```
But for practically every single Xorg package. They build fine without xorg-util-macros and pkg-config, but maybe we should add them as dependencies to everything? Thoughts?